frontend/src/components/ListDetail.js

```diff
@@ -43,12 +42,23 @@ export const ListDetail = ({ id, name, completed, items, handleUpdate }) => {
         setItemsState(newItems)
     }
 
-    const handleListUpdate = (e) => {
+    const handleListUpdate = async(e) => {
         if (e.charCode === 13) {
             handleUpdate(id, currentName)
         }
     }
 
+    const handleDeleteItem = async(listId, id) => {
+        const res = await deleteItem(listId, id);
+        if (res.error) {
+            console.log(res.error);
```

Comment:
Does this constitute a call to setErrorMessageState?

Reply:
changed console.log to setErrorMessageState